.github/workflows/main.yml

```diff
@@ -26,29 +26,33 @@ on:
       - "develop"
 
 jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.7"
```

Comment:
running precommit on different version of python than docker image won't cause any trobule?

Reply:
should not be an issue